src/main/java/com/bookbla/americano/domain/member/controller/dto/request/MemberProfileCreateRequest.java

```diff
@@ -14,25 +14,40 @@
 @AllArgsConstructor
 public class MemberProfileCreateRequest {
 
+    @NotNull(message = "이름이 입력되지 않았습니다.")
+    private String name;
+
+    @NotNull(message = "생년월일이 입력되지 않았습니다.")
+    private String birthDate;
+
+    @NotNull(message = "학교명이 입력되지 않았습니다.")
+    private String schoolName;
+
     @NotNull(message = "성별이 입력되지 않았습니다.")
     private String gender;
 
-    @NotNull(message = "생년월일이 입력되지 않았습니다.")
-    private LocalDate birthDate;
+    @NotNull(message = "프로필 사진이 입력되지 않았습니다.")
+    private String profileImageUrl;
 
-    @NotNull(message = "이름이 입력되지 않았습니다.")
-    private String name;
+    @NotNull(message = "카톡방 링크가 입력되지 않았습니다.")
+    private String openKakaoRoomUrl;
 
+    public LocalDate getBirthDate() {
+        return LocalDate.parse(birthDate);
+    }
 
-    @NotNull(message = "학교가 입력되지 않았습니다.")
-    private String schoolName;
+    public Gender getGender() {
+        return Gender.from(gender);
+    }
 
     public MemberProfileDto toDto() {
         return MemberProfileDto.builder()
             .gender(Gender.from(gender))
-            .birthDate(birthDate)
+            .birthDate(LocalDate.parse(birthDate))
```

Comment:
만들어놓은 메서드를 사용하거나 메서드를 지워도 될 것 같아요
```suggestion
            .birthDate(getBirthDate())
```